doc/computation.rst

```diff
@@ -143,20 +143,34 @@ name of the dimension as a key (e.g. ``y``) and the window size as the value
 
     arr.rolling(y=3)
 
-The label position and minimum number of periods in the rolling window are
-controlled by the ``center`` and ``min_periods`` arguments:
+Aggregation and summary methods can be applied directly to the ``Rolling``
+object:
 
 .. ipython:: python
 
-    arr.rolling(y=3, min_periods=2, center=True)
+    r = arr.rolling(y=3)
+    r.reduce(np.std)
+    r.mean()
 
-Aggregation and summary methods can be applied directly to the ``Rolling`` object:
+Aggregation results are at the end of each window by default, but can be
```

Comment:
"Aggregation results are assigned the coordinate at the end of each window by default"?

Reply:
Sorry, saw this, but got busy and forgot to come back to it until now.  Just force-pushed this change.